stats/stats.go

```diff
@@ -261,6 +261,20 @@ func (st *SampleTags) CloneTags() map[string]string {
 	return res
 }
 
+// CloudTags returns a SampleTags which is appropriate for cloud output.
+func (st *SampleTags) CloudTags() *SampleTags {
```

Comment:
Hmm this seems like the wrong place for this logic - it should be somewhere in `stats/cloud`

Reply:
Ah, but then you won't have access to the internal map :disappointed:

Reply:
hmm ok, after thinking about this some more, unfortunately this method probably has to be here, but even so, there are other issues:
- it modifies the original receiver (i.e. `st`), which is not clear from its name - it should be something like `ReplaceURLWithName()` and the comment should be the place that explains that we're using this for the cloud collector and a link to the issue
- it also doesn't make a lot of sense to return anything, since that might confuse a user of this method that we're returning a copy - it should be clear that this method modifies the internals of the current tag set
- I think we should check if `url` was actually present before modifying it - users might have ran their script with `k6 run --system-tags="name,method,status,error,check,group` (since these are the [basic requirements](https://github.com/loadimpact/k6/blob/c2d73ba070b5223d60e784401e9bda8d634440fe/stats/cloud/collector.go#L556))
- we should clear the json cache (`st.json`) if we modify anything, since that would invalidate it